.github/workflows/cncf-conformance.yaml

```diff
@@ -0,0 +1,146 @@
+        type: string
+
+env:
+  VERSION: ${{ github.event.inputs.version || 'latest' }}
+  REGISTRY: ${{ github.event.inputs.registry || 'ghcr.io/microshift-io' }}
+  TEST_TIMEOUT: ${{ github.event.inputs.test-timeout || '8400' }}
+
+jobs:
+  cncf-conformance:
+    strategy:
+      matrix:
+        runners: [ubuntu-24.04, ubuntu-24.04-arm]
+    name: Run CNCF conformance tests (${{ matrix.runners }})
+    runs-on: ${{ matrix.runners }}
+    steps:
+      - name: Check out MicroShift upstream repository
+        uses: actions/checkout@v4
+
+      - name: Detect CPU architecture
+        id: detect-cpu-arch
+        uses: ./.github/actions/arch
+
+      - name: Prepare the test environment
+        uses: ./.github/actions/prebuild
+
+      - name: Pull pre-built bootc image
+        shell: bash
+        run: |
+          set -euo pipefail
+
+          # Resolve version (use latest release if not specified)
+          if [ "${{ env.VERSION }}" = "latest" ]; then
+            VERSION=$(gh release list --limit 1 --json tagName --jq '.[0].tagName')
+            echo "Using latest release version: ${VERSION}"
+          else
+            VERSION="${{ env.VERSION }}"
+            echo "Using specified version: ${VERSION}"
+          fi
+
+          IMAGE="${{ env.REGISTRY }}/microshift"
+          TAG="${VERSION}-${{ steps.detect-cpu-arch.outputs.go_arch }}"
+
+          echo "Pulling ${IMAGE}:${TAG}"
+          sudo podman pull "${IMAGE}:${TAG}"
+          sudo podman tag "${IMAGE}:${TAG}" localhost/microshift-okd:latest
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: Setup 2-node cluster for CNCF tests
+        shell: bash
+        run: |
+          set -euo pipefail
+          make run
+          make run-ready
+          # Ensure first node is healthy before adding second node
+          make run-healthy
+          make add-node
+          # Ensure both nodes are healthy before running conformance tests
+          make run-healthy
+
+      - name: Configure cluster for CNCF conformance tests
+        shell: bash
+        run: |
+          set -euo pipefail
+
+          # Disable firewalld on cluster nodes to avoid blocking multi-node traffic
+          echo "Disabling firewalld on cluster nodes..."
+          for node in microshift-okd-1 microshift-okd-2; do
+            echo "  - Disabling firewalld on ${node}"
+            sudo podman exec "${node}" systemctl stop firewalld || true
+            sudo podman exec "${node}" systemctl disable firewalld || true
+          done
+
+          # Set default storage class for CNCF conformance tests
+          echo "Setting default storage class annotation..."
+          make env CMD="kubectl annotate storageclass topolvm-provisioner storageclass.kubernetes.io/is-default-class=true"
+
+          # Verify the annotation was applied
+          echo "Verifying default storage class:"
+          make env CMD="kubectl get storageclass topolvm-provisioner -o yaml | grep -A2 annotations"
```

Comment:
Please, open a bug for this. We need to set it by default in TopoLVM code

Reply:
https://github.com/microshift-io/microshift/issues/174